grammar.js

```diff
@@ -343,12 +344,52 @@ module.exports = grammar({
     },
 
     // LiteralString
-    string: ($) =>
-      seq(
-        field('start', alias($._string_start, 'string_start')),
-        field('content', optional(alias($._string_content, 'string_content'))),
-        field('end', alias($._string_end, 'string_end'))
-      ),
+    string: $ => choice($._string_literal, $._multiline_string_literal),
+
+    _string_literal: $ => choice(
```

Comment:
Found another issue:

```lua
local a = "a
b"
``` 

This is now valid. Althrough this is not a valid lua string.

Reply:
Gotcha, thanks! Will fix soon

Reply:
Are these valid lua escaped string?

`/u[0-9a-fA-F]{4}/`, `/U[0-9a-fA-F]{8}/` 

Something like this should be more appropriate? 🤔

```js
    escape_sequence: () =>
      token(
        prec(
          1,
          seq(
            '\\',
            choice(
              /[\nabfnrtv\\'"]/,
              /z\s*/,
              /[0-9]{1,3}/,
              /x[0-9a-fA-F]{2}/,
              /u\{[0-9a-fA-F]+\}/
            )
          )
        )
      ),
```

Reply:
Sorry for the delay last weeks of school was hectic, I've fixed \z and \090, bizarre cases in general